terragraph_planner/common/topology_models/test/helper.py

```diff
@@ -134,35 +134,35 @@ def raw_square_topology(demand: float = 0.025) -> Topology:
             site_id="DN1",
             site_type=SiteType.DN,
             location=GeoLocation(
-                utm_x=160, utm_y=510, utm_epsg=32631, altitude=0
```

Comment:
Why do the coordinates and azimuths need to change in this file? It seems that the frequency unit conversion should only affect the values right?

Reply:
The fix changes the capacity calculations (generally reducing them). This topology was originally added for unit testing so that all links would have 1.8 Gbps throughput. The change in the rain rate formula dropped the MCS class of some of the links. This is addressed by simply scaling down the unit test topology by 10%. Then, all the links are 1.8 Gbps again.